ci/run_locally.sh

```diff
@@ -62,8 +63,19 @@ if [ "${interactive_mode}" == 'true' ];then
     if [ "${workflow}" == 'QUIT' ]; then
       break
     else
+      declare -a opt_args
+      env_file="${conf_file_location}/${workflow}.vars"
+      if [ -f "${env_file}" ]; then
+        log_debug "\n=== Using envs from ${env_file} ==="
+        opt_args+=(--var-file "${env_file}")
+      fi
+      inputs_file="${conf_file_location}/${workflow}.inputs"
+      if [ -f "${inputs_file}" ]; then
+        log_debug "\n=== Using inputs from ${env_file} ==="
+        opt_args+=(--input-file "${inputs_file}")
+      fi
       log_debug "\n=== Running ${workflows_dir}/${workflow}.yml workflow ==="
-      act --detect-event --rm -W "${workflows_dir}/${workflow}.yml" || fn_die "ERROR: attempt to run ${workflows_dir}/${workflow}.yml workflow locally has failed. Exiting ..."
+      act "${opt_args[@]}" --detect-event --rm -W "${workflows_dir}/${workflow}.yml" || fn_die "ERROR: attempt to run ${workflows_dir}/${workflow}.yml workflow locally has failed. Exiting ..."#+
```

Comment:
What is `#+` in the end of the line for :thinking:

Reply:
... left over.... :smile:

Reply:
thx